protos/chat.proto

```diff
@@ -4,7 +4,7 @@ package chat;
 
 import "google/protobuf/timestamp.proto";
 
-option go_package="github.com/rodaine/grpc-chat/protos";
+option go_package="./protos";
```

Comment:
So we want to keep this the same as it drives what dependent code would use when generating import statements in go. You should be able to modify the `go_out` and `go-grpc_out` options passed to protoc to use the same directory path with [`paths=source_relative`](https://developers.google.com/protocol-buffers/docs/reference/go-generated#invocation)